react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+
+export function StopWatch() {
+  const [seconds, setSeconds] = useState(0);
+  const [play, setPlay] = useState(false);
+  const [pauseID, setPauseID] = useState(null);
+
+  function startTimer() {
+    if (!play) {
+
+      const count = setInterval(() => {
+        setPlay(true);
+        setPauseID(count);
+        setSeconds((seconds) => seconds + 1);
+      }, 1000);
+    }
+    playPause(pauseID);
+  }
+
+  function playPause() {
+    clearInterval(pauseID);
```

Comment:
When you clear the interval, `pauseID` becomes invalid. You should unset your state.